src/test/java/nextstep/subway/utils/Request.java

```diff
@@ -0,0 +1,43 @@
+package nextstep.subway.utils;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+
+import java.util.Map;
+
+public class Request {
+
+    public static <T> ExtractableResponse<Response> post(final String url, final T requestBody) {
+        return RestAssured
+                .given().log().all()
+                .body(requestBody)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post(url)
+                .then().log().all()
+                .extract();
+    }
+
+    public static ExtractableResponse<Response> get(final String url, Map<String, String> params) {
+        return RestAssured.given().log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .get(url + createQueryParams(params))
```

Comment:
```suggestion
                .queryParams(params)
                .get(url)
```
queryParams라는 메서드가 제공되니 참고하세요